src/Microsoft.TestPlatform.AdapterUtilities/ManagedNameUtilities/InvalidManagedNameException.cs

```diff
@@ -20,7 +18,7 @@ public class InvalidManagedNameException :
     , ISerializable
 #endif
 {
-    public InvalidManagedNameException(string message) : base(message) { }
+    public InvalidManagedNameException(string? message) : base(message) { }
```

Comment:
I think the `message` is never null, and it shouldn't be. If we are throwing this, we should report to user what's wrong.

Reply:
All basic .net exceptions allow null message that's why I am allowing null too.